package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/JuanImbachi/protractor-workshop-2020#readme",
   "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2",
```

Comment:
This dependency should be installed as dev-dependency (`npm install --save-dev`)

Reply:
![image](https://user-images.githubusercontent.com/41926056/88469540-0c118300-ceb8-11ea-9646-6430032663e6.png)
I already fixed. I post again the screenshot of the test, in case you need it.